test/mock_dl/index.ts

```diff
@@ -47,7 +47,8 @@ app.post('/mock/tokens/generate', (req, res) => {
     const token = get_token(req);
 
     res.send({
-        conversationId,
+        // We don't have a working mock on token generate/refresh yet
+        conversationId: 'DUMMY',
```

Comment:
probably won't pass poliCheck :) can you change these?

Reply:
Done.